src/NuGet.Core/NuGet.ProjectModel/LockFile/LockFileUtilities.cs

```diff
@@ -26,5 +27,20 @@ public static LockFile GetLockFile(string lockFilePath, Common.ILogger logger)
 
             return lockFile;
         }
+
+        public static LockFile SafeGetLockFile(string lockFilePath, Common.ILogger logger)
```

Comment:
why do we need a different function here? why not just make the original GetLockFile to use SafeRead?